web/src/main/webapp/xslt/skin/default/skin.xsl

```diff
@@ -186,7 +188,7 @@
                   select="if (util:getUiConfigurationJsonProperty(/root/request/ui, 'mods.footer.enabled') = 'false')
                         then false()
                         else true()"/>
-    <xsl:if test="$isFooterEnabled">
+    <xsl:if test="$isFooterEnabled and $output != 'pdf'">
```

Comment:
Would it not be more efficient to do the following?
```
<xsl:template name="footer">
  <xsl:if test="$output != 'pdf'">
    <!-- Footer goes here -->
  </xsl:if>
</xsl:template>
```

Reply:
Not really sure if would make a difference in the speed to create the PDF, but I've committed a change to optimise this.